sulis_state/src/merchant_state.rs

```diff
@@ -134,7 +138,12 @@ impl MerchantState {
     }
 
     pub fn get_sell_price(&self, item_state: &ItemState) -> i32 {
-        ((item_state.item.value as f32) * self.sell_frac).floor() as i32
+        let value = if item_state.identified {
+            item_state.item.value as f32
+        } else {
+            (1.0 - self.identify_fraction) * item_state.item.value.abs() as f32
```

Comment:
I'm not sure the player should really know the value of an identified item before it is identified.  I think in a lot of games an unidentified item sells for a nominal amount, for example the value of the base item without any bonuses.

But I think this is probably ok as-is for a first pass.

Reply:
Agreed.